src/main/scala/com/microsoft/azure/sqldb/spark/sql/streaming/SQLWriter.scala

```diff
@@ -0,0 +1,193 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Please review the license headers in the source files to ensure consistency with the rest of the repo (ASF vs. MIT)